app/src/main/java/com/intervaltimer/google/activities/WorkoutActivity.java

```diff
@@ -86,6 +88,8 @@
     private TimerService timerService = null;
     private boolean serviceBound = false;
 
+    private final String WORKOUT_INTENT_TAG = "INTENT";
```

Comment:
I already asked it before: how do they currently handle logging? is your "style" consistent with their current approach?

E.g., I often see code that uses the class name as TAG...